src/elections/crud.py

```diff
@@ -0,0 +1,68 @@
+import dataclasses
+import logging
+from datetime import datetime
+
+import database
+import sqlalchemy
+from elections.tables import Election
+from officers.constants import OfficerPosition
+from officers.tables import OfficerInfo, OfficerTerm
+
+_logger = logging.getLogger(__name__)
+
+async def get_election(db_session: database.DBSession, election_slug: str) -> Election | None:
+    query = sqlalchemy.select(Election)
+    query = query.where(Election.slug == election_slug)
+    result = (await db_session.execute(query)).scalar()
+    db_session.commit()
+    return result
+
+async def create_election(params: dict[str, datetime], db_session: database.DBSession) -> None:
```

Comment:
We should probably make a dataclass out of this `dict`, to be a little more strongly typed?